js/views/transactionsVw.js

```diff
@@ -281,6 +285,118 @@ module.exports = baseVw.extend({
     tabWrapper.append(orderShort.render().el);
   },
 
+  clickDownloadCSV: function(e){
+    var targBtn = $(e.target);
+    targBtn.closest('.btn').addClass('loading');
+    this.downloadCSV(targBtn);
+  },
+
+  downloadCSV: function(targBtn){
+    var self = this,
+        rawData = [],
+        calls = [],
+        dataCSV = '',
+        tempAnchor = document.createElement('a'),
+        dataBlob,
+        exportData = function(data){
+          "use strict";
```

Comment:
I'm guessing this might be a copy & paste thing, but can we please stop putting 'use strict' at the function level? It can be put at the top of the page and that would target the entire page.